electron/src/schema-salad-resolver/schema-salad-resolver.ts

```diff
@@ -121,18 +126,23 @@ function traverse(data, source, root, rootPath, graph = {}, traversedExternalPat
                                 Object.assign(data, newData);
 
                                 break;
+                            /**
+                             * When embedding a “run” value, we need to store the original source of that content so we can later remove
+                             * the content and restore the path to it
+                             */
                             case "run":
                             default:
                                 Object.assign(data, {
                                     [key]: content,
                                     "sbg:rdfId": entry,
                                     "sbg:rdfSource": externalPath
                                 });
+                                console.log("After run parsed", data);
```

Comment:
Assuming the console.logs are left here for a reason?

Reply:
fixed